cmd/snap/cmd_snap_op.go

```diff
@@ -202,18 +203,24 @@ func refreshAll() error {
 	if err != nil {
 		return fmt.Errorf("cannot list updates: %s", err)
 	}
+	// nothing to update/list
+	if len(updates) == 0 {
+		return nil
```

Comment:
I looked at the code and we currently do the following when we find nothing:
- list: prints `No snaps are installed yet. Try 'snap install hello-world'.` (fixes https://bugs.edge.launchpad.net/ubuntu/+bug/1587445 from mark)
- find: prints `no snaps found for %q"`

this makes me wonder if maybe we should print: `all snaps are fresh` or something similar to be consistent. Or could we undo the fix for LP: #1587445 and remove the find print too? Anyway, sorry for dragging this on but I think since this review the fix for LP: #1587445 which raises this new question.